components/logicbutton/stories/logicbutton.stories.js

```diff
@@ -32,4 +32,7 @@ export default {
 };
 
 export const Default = Template.bind({});
-Default.args = {};
+Default.args = {
+  variant: "and"
```

Comment:
```suggestion
```

Since this matches the default value set in the args above, you don't necessarily need to set it here as well.  Up to you!

Reply:
removed it!